test/fixtures/win.js

```diff
@@ -1767,6 +1839,24 @@ export const escape = {
         expected: '\\^"a',
       },
     ],
+    "double quotes ('\"') + backslashes ('\\')": [
+      {
+        input: 'a\\"b',
+        expected: 'a\\\\\\^"b',
+      },
+      {
+        input: 'a\\"b\\"c',
+        expected: 'a\\\\\\^"b\\\\\\^"c',
+      },
+      {
+        input: 'a\\"',
+        expected: 'a\\\\\\^"',
+      },
+      {
+        input: '\\"a',
+        expected: '\\\\\\^"a',
+      },
+    ],
```

Comment:
These 4 test cases should have existed before this change. If this Pull Request is rejected these test cases should be added to the project separately.

Reply:
Resolved by  #2034